src/Sarif.FunctionalTests/ConverterTestData/AndroidStudio/AndroidStudio_issueLog5_raw.xml.sarif

```diff
@@ -53,10 +53,6 @@
           "message": "Typo: In word 'userguide'",
           "locations": [
             {
-              "analysisTarget": {
-                "uri": "gradle.properties",
-                "uriBaseId": "$PROJECT_DIR$"
-              },
```

Comment:
Yes! Since they're both the same, we don't need to populate `result.analysisTarget`.